source/air/src/profiler.rs

```diff
@@ -89,7 +104,91 @@ impl Profiler {
         user_quant_costs.sort_by_key(|v| v.instantiations * v.cost);
         user_quant_costs.reverse();
 
-        Ok(Profiler { message_interface, quantifier_stats: user_quant_costs })
+        Ok(Profiler { message_interface, quantifier_stats: user_quant_costs, instantiation_graph })
+    }
+
+    pub fn instantiation_graph(&self) -> Option<&InstantiationGraph> {
+        self.instantiation_graph.as_ref()
+    }
+
+    fn make_instantiation_graph(model: &Model) -> InstantiationGraph {
```

Comment:
I'm just looking at this now, but `make_instantiation_graph` seems a bit duplicative with the graph construction in the smt2utils crate.  Should we consolidate the logic in one place or the other?